### DISCUSSION TITLE
Firebase analytics not logging anything

### DISCUSSION THREAD

**Discussion by brookmg** [invertase/react-native-firebase#6330] [Q&A]
I followed the documentation on how to add firebase analytics on a react native app but no data is being displayed on the dashboard. I implemented this in android/build.gradle in case it's a version mismatch,

```maven
project.ext {
    set('react-native', [
        versions: [
            // Overriding Build/Android SDK Versions
            android : [
                minSdk: 21,
                targetSdk: 31,
                compileSdk: 31,
                buildTools: "30.0.2"
            ],

            // Overriding Library SDK Versions
            firebase: [
                // Override Firebase SDK Version
                bom: "30.1.0"
            ],
        ],
    ])
}
```

// package.json
```
"@react-native-firebase/analytics": "^14.11.0",
"@react-native-firebase/app": "^14.11.0",
"@react-native-firebase/app-check": "^14.11.0",
"@react-native-firebase/auth": "^14.11.0",
"@react-native-firebase/crashlytics": "^14.11.0",
"@react-native-firebase/messaging": "^14.11.0",
"@react-native-firebase/perf": "^14.11.0",
"@react-native-firebase/remote-config": "^14.11.0",
```

**Comment by mikehardy** [comment]
Hi there!
Not sure what to say - unfortunately there may not be much to say here, but you should enable debug via adb commands as specified here: https://firebase.google.com/docs/analytics/get-started?platform=android#start_logging_events_2

The typical cause for this is adblock programs somewhere in your network path to google (on your device or wifi router or similar)

I can state with 100% certainty that the analytics code *is* working in this module and in the SDK. There are zero reports of failures when properly set up (and not blocked) and I see it personally work in my work projects. So this will be project-specific or device-specific or network-specific for you which means we won't have much to offer here beyond the information in this comment
